apps/omg_watcher_rpc/test/omg_watcher_rpc/web/controllers/utxo_test.exs

```diff
@@ -16,7 +16,7 @@ defmodule OMG.WatcherRPC.Web.Controller.UtxoTest do
   use ExUnitFixtures
   use ExUnit.Case, async: false
   use OMG.Fixtures
-  use OMG.Watcher.Fixtures
+  use OMG.WatcherInformational.Fixtures
```

Comment:
nitpick: `WatcherInformational` is quite a long module name 🤔

I don't have better names though other than `WatcherData` or `WatcherInfo`

Reply:
much :+1: on `WatcherInfo`. `WatcherData` can be a bit confusing

Reply:
Going with `WatcherInfo`